aldryn_reversion/admin.py

```diff
@@ -102,14 +108,26 @@ def post_move_plugin(self, request, source_placeholder, target_placeholder,
             request, source_placeholder, target_placeholder, plugin)
         comment_dict = self.get_commen_plugin_info(plugin)
         comment = _('Moved plugin #%(plugin_id)s: %(plugin)s') % comment_dict
-        self._create_revision(plugin, request.user, comment)
+        self._create_aldryn_revision(plugin, request.user, comment)
 
     def post_delete_plugin(self, request, plugin):
         super(VersionedPlaceholderAdminMixin, self).post_delete_plugin(
             request, plugin)
         comment_dict = self.get_commen_plugin_info(plugin)
         comment = _('Deleted plugin #%(plugin_id)s: %(plugin)s') % comment_dict
-        self._create_revision(plugin, request.user, comment)
+        self._create_aldryn_revision(plugin, request.user, comment)
+
+    def get_revision_data(self, request, object):
+        """
+        Backward compatible method to get revision data. Returns list of objects
+        that should be used to create revision.
+        if django-reversion < 1.9.0 calls to super method since it exists,
+        if django-reversion >= 1.9.0 returns [object].
+        """
+        if DJANGO_REVERSION_VERSION < LooseVersion('1.9.0'):
+            return super(VersionedPlaceholderAdminMixin,
+                         self).get_revision_data(request, object)
+        return [object]
```

Comment:
actually this is wrong, previous get reversion data was returning a dict of data to be used in revision, OR return none, since maybe latest version of reversions _create_revision does that.... needs to be investigated.